Makefile

```diff
@@ -1806,6 +1806,7 @@ fuzz: bin/fuzz
 bazel-generate: ## Generate all bazel BUILD files.
 	@echo 'Generating DEPS.bzl and BUILD files using gazelle'
 	./build/bazelutil/bazel-generate.sh
+	rm -f bazel-out # even with --symlink_prefix this still gets created somehow
```

Comment:
making it and then deleting it could still cause file-watching tools to go nuts indexing all the files they (briefly) saw. I might instead say we rm it and touch it as a regular file _first_ so it never exists as a symlink?

Reply:
The existence of the `bazel-out` symlink is really convenient for a variety of day-to-day tasks when you're using Bazel as your build system (which I understand `cockroach` developers aren't doing today, but it's supposed to be a matter of time anyway). `rm`'ing it unconditionally seems like throwing the baby out with the bathwater.

Reply:
The prefix’ed sum link is still there fwiw.

I think letting the user decide to touch it as a regular file (or not if they prefer) is fine — we don’t need to `rm` it.

Reply:
will be `_bazel/out` now.